third_party/gpus/sycl_configure.bzl

```diff
@@ -381,7 +409,33 @@ def _create_local_sycl_repository(repository_ctx):
     ]}
 
     bash_bin = get_bash_bin(repository_ctx)
-    sycl_config = _get_sycl_config(repository_ctx, bash_bin)
+   
+    hermetic = repository_ctx.getenv("SYCL_BUILD_HERMETIC") == "1"
+    if hermetic:
+        oneapi_version = repository_ctx.getenv("ONEAPI_VERSION") or ""
```

Comment:
You can replace it with `repository_ctx.getenv("ONEAPI_VERSION", "")` here and in other places where you need a default value different from `None`.

Reply:
Thank you @ybaturina, I have made the above changes